src/main/java/seedu/address/logic/commands/DeleteTransactionCommand.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Transaction;
+
+/**
+ * Deletes a transaction from a person's transaction list using its specified index.
+ */
+public class DeleteTransactionCommand extends Command {
+    public static final String COMMAND_WORD = "deletet";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the transaction identified by the index number used in the displayed transaction list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_TRANSACTION_SUCCESS = "Deleted transaction %1$s\\nfrom %2$s";
+
+    private static final Index CURRENT_PERSON = Index.fromOneBased(1);
```

Comment:
I realised for your logic to work, we need the person list to be filtered when showing the transaction list. I will make an update on that.

Reply:
PR #71 is up to fix this. Should be alright after merging.

Reply:
Just merged this PR. Could you pull the updates and see if `deletet` works?